src/java/voldemort/tools/admin/command/AdminCommandAsyncJob.java

```diff
@@ -182,12 +183,18 @@ public static void executeCommand(String[] args) throws IOException {
         public static void doAsyncJobList(AdminClient adminClient, List<Integer> nodeIds) {
             // Print the job information
             for(Integer nodeId: nodeIds) {
-                System.out.println("Retrieving async jobs from node " + nodeId);
+                System.out.print("Retrieving async jobs from node " + nodeId);
                 List<Integer> asyncIds = adminClient.rpcOps.getAsyncRequestList(nodeId);
-                System.out.println("Async Job Ids on node " + nodeId + " : " + asyncIds);
+                System.out.println(" : " + asyncIds);
                 for(int asyncId: asyncIds) {
-                    System.out.println("Async Job Id " + asyncId + " ] "
-                                       + adminClient.rpcOps.getAsyncRequestStatus(nodeId, asyncId));
+                    System.out.print("\tAsync Job Id " + asyncId + " : ");
+                    try {
+                        AsyncOperationStatus status = adminClient.rpcOps.getAsyncRequestStatus(nodeId, asyncId);
+                        System.out.println(status);
+                    } catch (Exception e) {
+                        System.out.println("threw " + e.getClass().getSimpleName() + "!");
```

Comment:
The message is included as part of the stacktrace, which is printed on the line below.